gprofiler/client.py

```diff
@@ -113,7 +124,13 @@ def get_health(self):
         return self.get("health_check")
 
     def submit_profile(
-        self, start_time: datetime.datetime, end_time: datetime.datetime, profile: str, total_samples: int
+        self,
+        start_time: datetime.datetime,
+        end_time: datetime.datetime,
+        profile: str,
+        total_samples: int,
+        spawn_time: float,
+        metrics: 'Metrics',
```

Comment:
```suggestion
        metrics: Metrics,
```
and add `from __future__ import annotations`

Reply:
We officially support Python 3.6, which doesn't have this feature

Reply:
Oof